### ISSUE TITLE
The performance of graph classification on the D&D dataset.

### ISSUE THREAD

**Issue by hjlin0515** [HongyangGao/gunet#6]
I have ran the "run_GUNet.sh DD 0" for many times and the graph classification accuracy only reach 81.2% (The corresponding result reported in the paper is 82.43%). Is there anything wrong in the hyper-parameter settings or something else? And It seems normal in other two datasets. Thank you.

**Comment by HongyangGao** [comment]
Hi Haojie,

I run the code on DD. The performance is about 82.0. I think you can
increase the drop rate of GCN to 0.35 in ops.py line 117.

Best,

Hongyang

**Comment by hjlin0515** [comment]
Thank you very much for your answer.
I increase the the dropout rate to 0.35. After many attempts, the performance is about 81.4. 
And is it appropriate to choose the best acc in all epochs as the final testing result? It seems that the testing result is not much stable.

**Comment by HongyangGao** [comment]
Hi,

I am sorry for the results. I will check it again. The performance report
method is widely used such as GIN. The graph data are not as large as
ImageNet which can provide stable performance evaluation. So the community
currently select the best average performance for 10-fold experiments.

Best,

Hongyang

**Comment by hjlin0515** [comment]
Thank you for your explanation!

**Comment by hjlin0515** [comment]
Hi, 
Does the "best average performance for 10-fold" means average the 10-fold performance in different iterations and choose the best performance among overall iterations(like [DiffPool](https://github.com/RexYing/diffpool/blob/8e2a95fa1f31e8fb31cf2fa25f3d639904885e28/train.py#L521) the method you compared) ? 
However, in your code, you choose the best performance in each fold first and then average these best performance as the evaluation metric. And I can not find the the code of  corresponding computing in the GIN code, can you provide the link? Thank you very much!
